lighthouse-core/test/lib/traces/tracing-processor-test.js

```diff
@@ -47,25 +47,6 @@ describe('TracingProcessor lib', () => {
     });
   });
 
-  it('doesn\'t throw when user_timing events have a colon', () => {
```

Comment:
wait not nevermind it does hurt something there's no init function at all anymore :)